pkg/v2/rrsetManager.go

```diff
@@ -0,0 +1,186 @@
+
+	// RRSet is list of records grouped by their name and type.
+	RRSet struct {
+		UUID      string       `json:"uuid"`
+		ZoneUUID  string       `json:"zone"` //nolint: tagliatelle
+		Name      string       `json:"name"`
+		TTL       int          `json:"ttl"`
+		Type      RecordType   `json:"type"`
+		Comment   string       `json:"comment"`
+		ManagedBy string       `json:"managed_by"`
+		Records   []RecordItem `json:"records"`
+	}
+
+	// RecordItem represents single record from RRSet.
+	RecordItem struct {
+		Content  string `json:"content"`
+		Disabled bool   `json:"disabled"`
+	}
+
+	rrsetCreationForm struct {
+		Name      string       `json:"name"`
+		TTL       int          `json:"ttl"`
+		Type      RecordType   `json:"type"`
+		Records   []RecordItem `json:"records"`
+		Comment   string       `json:"comment,omitempty"`
+		ManagedBy string       `json:"managed_by,omitempty"`
+	}
+
+	rrsetUpdateForm struct {
+		TTL       int          `json:"ttl"`
+		Records   []RecordItem `json:"records"`
+		Comment   string       `json:"comment,omitempty"`
+		ManagedBy string       `json:"managed_by,omitempty"`
+	}
+
+	rrsetManagedByForm struct {
+		ManagedBy string `json:"managed_by"`
+	}
+)
+
+func (s *RRSet) CreationForm() (io.Reader, error) {
+	form := rrsetCreationForm{
+		Name:      s.Name,
+		TTL:       s.TTL,
+		Type:      s.Type,
+		Records:   s.Records,
+		Comment:   s.Comment,
+		ManagedBy: s.ManagedBy,
+	}
+	body, err := json.Marshal(form)
+
+	return bytes.NewReader(body), err
+}
+
+func (s *RRSet) UpdateForm() (io.Reader, error) {
+	form := rrsetUpdateForm{
+		TTL:       s.TTL,
+		Records:   s.Records,
+		Comment:   s.Comment,
+		ManagedBy: s.ManagedBy,
+	}
+	body, err := json.Marshal(form)
+
+	return bytes.NewReader(body), err
+}
+
+// CreateRRSet request to create a new rrset for the zone.
+func (c *Client) CreateRRSet(ctx context.Context, zoneID string, rrset Creatable) (*RRSet, error) {
+	form, err := rrset.CreationForm()
+	if err != nil {
+		return nil, fmt.Errorf("rrset creation form: %w", err)
+	}
+	r, e := c.prepareRequest(
+		ctx, http.MethodPost, fmt.Sprintf(rrsetPath, zoneID), form, nil, nil,
+	)
+
+	return processRequest[RRSet](c.httpClient, r, e)
+}
+
+// DeleteRRSet request to delete the rrset from zone by zoneID and rrsetID.
+func (c *Client) DeleteRRSet(ctx context.Context, zoneID, rrsetID string) error {
+	r, e := c.prepareRequest(
+		ctx, http.MethodDelete, fmt.Sprintf(singleRRSetPath, zoneID, rrsetID), nil, nil, nil,
+	)
+	_, err := processRequest[RRSet](c.httpClient, r, e)
+
+	return err
+}
+
+// GetRRSet returns a single rrset from zone by zoneID and rrsetID.
+func (c *Client) GetRRSet(ctx context.Context, zoneID, rrsetID string) (*RRSet, error) {
+	r, e := c.prepareRequest(
+		ctx, http.MethodGet, fmt.Sprintf(singleRRSetPath, zoneID, rrsetID), nil, nil, nil,
+	)
+
+	return processRequest[RRSet](c.httpClient, r, e)
+}
+
+// ListRRSets returns a list of rrsets by zoneID and options.
+// non necessarily options:
```

Comment:
There is no need in that description since, it's just wrapper and all documentation expected to be in DNSv2 public api documentation.  
I believe that it's just an excessive point of doc that will never be updated, if update occur.